src/App.tsx

```diff
@@ -17,18 +66,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterType={filterType}
+                setFilterType={setFilterType}
+                query={query}
+                setQuery={setQuery}
+                applyQuery={applyQuery}
+                cleanQuery={setAppliedQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoader && <Loader />}
+
+              {(todos.length) && (
```

Comment:
```suggestion
              {(!!todos.length) && (
```
to avoid printing 0 in DOM